esmvaltool/preprocessor/_derive/rsnst.py

```diff
@@ -0,0 +1,45 @@
+"""Derivation of variable `rsnst`.
+
+authors:
+    - weig_ka
+
+"""
+from iris import Constraint
+
+from ._baseclass import DerivedVariableBase
+
+
+class DerivedVariable(DerivedVariableBase):
+    """Derivation of variable `rsnst`."""
+
+    # Required variables
+    required = [
```

Comment:
Please, make it a bit more concise (see lvp)

Reply:
OK, see above